frontend/src/component/admin/network/NetworkTrafficUsage/chart-functions.ts

```diff
@@ -40,6 +42,46 @@ export const toChartData = (
     return { datasets, labels };
 };
 
+export const toConnectionChartData = (
+    traffic: TrafficUsageDataSegmentedCombinedSchema,
+): { datasets: ChartDatasetType[]; labels: string[] } => {
+    const { newRecord, labels } = getLabelsAndRecords(traffic);
+    const datasets = traffic.apiData
+        .sort(
+            (item1, item2) =>
+                endpointsInfo[item1.apiPath].order -
+                endpointsInfo[item2.apiPath].order,
+        )
+        .map((item) => {
+            const record = newRecord();
+            for (const dataPoint of Object.values(item.dataPoints)) {
+                const date = parseISO(dataPoint.period);
+                const requestCount = dataPoint.trafficTypes[0].count;
+
+                if (dataPoint.period.length === 7) {
```

Comment:
if YYYY-MM then divide requests by 1 day connection (7200 req) and number of days in a month

Reply:
```suggestion
                if (traffic.grouping === 'monthly') {
```